monasca/values.yaml

```diff
@@ -64,6 +64,8 @@ influxdb:
   config:
     http:
       bind_address: 8086
+    data:
+      cache_max_memory_size: "1073741824"
```

Comment:
for reference, this fixes rounding errors when diff'd by landscaper - cache_max_memory_size is an int64 but if the chart gets converted to json the number will be rounded. this can cause tooling like landscaper to try to redeploy monasca every time any component is updated